src/views/shared/SecurityBeacon.js

```diff
@@ -28,14 +43,19 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
     el.css('background-image', '');
     el.removeClass('new-user undefined-user');
     if (isNewUser) {
+      el.find('.auth-beacon-description').text('no security image');
```

Comment:
I decided to leave this out based on the lack of localization and also the absence of a request from the customer.